k8s/upgrades/0.9.0-1.0.0/cstor_pool_upgrade.sh

```diff
@@ -40,7 +42,8 @@ function verify_openebs_version() {
 
     if [[ $openebs_version != $current_version ]] && \
           [[ $openebs_version != $pool_upgrade_version ]]; then
-        echo "Expected version of $name_res in $resource is $current_version but got $openebs_version";exit 1;
+        echo "Expected version of $name_res in $resource is $current_version but got $openebs_version"
```

Comment:
do error_msg here as well

Reply:
Not required caller of this function is handling